packages/discovery-styles/scripts/update-styles.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+PDFJS_WEB_CSS=../../node_modules/pdfjs-dist/web/pdf_viewer.css
+PDFJS_SCSS=scss/components/document-preview/_pdfjs_web_mixins.scss
+
+function replace_quote() {
+  file=$1
+  key=$2
+  tmp=$file.tmp
+  
+  sed -e "/BEGIN-QUOTE $key/q" $file > $tmp
+  cat >> $tmp
+  sed -ne "/END-QUOTE $key/,\$p" $file >> $tmp
+  cp $tmp $file;
+  rm $tmp;
+}
+
+cat $PDFJS_WEB_CSS | awk '/^\/\*/,/\*\//' | replace_quote $PDFJS_SCSS "COMMENT"
+cat $PDFJS_WEB_CSS | awk '/^\.textLayer/,/}/' | replace_quote $PDFJS_SCSS "TEXT-LAYER"
```

Comment:
I'm a bit concerned about using string manipulation for these style script files. There's existing utilities (like `eslint`/`prettier`) which will correctly read in the files and make them available to manipulation in a safe way. Writing a plugin for such a utility would be better.

Reply:
OK. I'd use CSS parser (`postcss`) and its AST to write the script.